sql/database.sql

```diff
@@ -107,13 +107,16 @@ CREATE TABLE `api_log` (
 
 DROP TABLE IF EXISTS `api_token`;
 CREATE TABLE `api_token` (
-  `id` bigint(20) NOT NULL AUTO_INCREMENT,
-  `token_api` varchar(4),
-  `user_id` bigint(20) NOT NULL,
+  `id` bigint(20) NOT NULL,
+  `user_id` varchar(40) DEFAULT NULL,
+  `token` varchar(128) DEFAULT NULL,
+  `expiry` datetime DEFAULT NULL,
+  `token_auth` text,
+  `token_api` varchar(4) DEFAULT NULL,
   `patient_id` bigint(20) NOT NULL,
-  `token` varchar(40) DEFAULT NULL,
-  `token_auth` varchar(255),
-  `expiry` datetime NULL,
+  `client_id` varchar(80) DEFAULT NULL,
+  `auth_user_id` varchar(80) DEFAULT NULL,
+  `scope` text COMMENT 'json encoded'
```

Comment:
actually, the error is prob from missing comma at end of above line

Reply:
don't know what happen here. tired I suppose